proto/kvrpcpb.proto

```diff
@@ -487,12 +487,14 @@ message MvccGetByStartTsResponse {
 message SplitRegionRequest {
     Context context = 1;
     bytes split_key = 2;
```

Comment:
Maybe we can change it `repeated` instead of adding a new filed.

Reply:
em, seems not a good idea. If new tidb connects to an old tikv, tikv will silently split the region into 2 with no error.

Reply:
Could you mark it deprecated?

Reply:
good idea

Reply:
done @overvenus